tools/bpftool/bpftool.vcxproj

```diff
@@ -279,4 +279,4 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
-</Project>
+</Project>
```

Comment:
Why this change?

Reply:
Editor stripped new lines from end of file for some reason.